medicines/doc-index-updater/src/storage_client/mod.rs

```diff
@@ -1,15 +1,54 @@
-use azure_sdk_core::errors::AzureError;
-use azure_sdk_storage_core::prelude::Client;
+use async_trait::async_trait;
+use azure_sdk_core::{errors::AzureError, prelude::*, DeleteSnapshotsMethod};
+use azure_sdk_storage_blob::Blob;
+use azure_sdk_storage_core::prelude::*;
```

Comment:
do we have to have `prelude::*` or can we be explicit with `Client` etc? It's not clear when we use `Client` from where we get it